From cac1fcf7a66c3bbecbcf66851d59795163d2e444 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ren=C3=A9=20Stadler?= <rene.stadler@nokia.com>
Date: Wed, 29 Jun 2011 19:50:42 +0300
Subject: [PATCH] pulsesink: remove unused member variable and misleading log message

Wim changed it in commit 8bfd80 so that pa_defer_ran is not read
anywhere.

The log message used to annotate a mainloop_wait call which is gone.
---
 ext/pulse/pulsesink.c |    5 -----
 ext/pulse/pulsesink.h |    2 --
 2 files changed, 0 insertions(+), 7 deletions(-)

diff --git a/ext/pulse/pulsesink.c b/ext/pulse/pulsesink.c
index 9fe46e3..8f9ba70 100644
--- a/ext/pulse/pulsesink.c
+++ b/ext/pulse/pulsesink.c
@@ -1057,7 +1057,6 @@ mainloop_enter_defer_cb (pa_mainloop_api * api, void *userdata)
   gst_element_post_message (GST_ELEMENT (pulsesink), message);
 
   /* signal the waiter */
-  pulsesink->pa_defer_ran = TRUE;
   pa_threaded_mainloop_signal (mainloop, 0);
 }
 
@@ -1074,7 +1073,6 @@ gst_pulseringbuffer_start (GstRingBuffer * buf)
   pa_threaded_mainloop_lock (mainloop);
 
   GST_DEBUG_OBJECT (psink, "scheduling stream status");
-  psink->pa_defer_ran = FALSE;
   pa_mainloop_api_once (pa_threaded_mainloop_get_api (mainloop),
       mainloop_enter_defer_cb, psink);
 
@@ -1134,7 +1132,6 @@ mainloop_leave_defer_cb (pa_mainloop_api * api, void *userdata)
   gst_message_set_stream_status_object (message, &val);
   gst_element_post_message (GST_ELEMENT (pulsesink), message);
 
-  pulsesink->pa_defer_ran = TRUE;
   pa_threaded_mainloop_signal (mainloop, 0);
   gst_object_unref (pulsesink);
 }
@@ -1182,12 +1179,10 @@ cleanup:
   }
 
   GST_DEBUG_OBJECT (psink, "scheduling stream status");
-  psink->pa_defer_ran = FALSE;
   gst_object_ref (psink);
   pa_mainloop_api_once (pa_threaded_mainloop_get_api (mainloop),
       mainloop_leave_defer_cb, psink);
 
-  GST_DEBUG_OBJECT (psink, "waiting for stream status");
   pa_threaded_mainloop_unlock (mainloop);
 
   return res;
diff --git a/ext/pulse/pulsesink.h b/ext/pulse/pulsesink.h
index 0f75fcc..e3cbbca 100644
--- a/ext/pulse/pulsesink.h
+++ b/ext/pulse/pulsesink.h
@@ -64,8 +64,6 @@ struct _GstPulseSink
   gboolean mute:1;
   gboolean mute_set:1;
 
-  gboolean pa_defer_ran:1;
-
   gint notify; /* atomic */
 
   const gchar *pa_version;
